modules/persistent-storage-csi-efs-sts.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+
+:_content-type: PROCEDURE
+[id="efs-sts_{context}"]
+= Configuring AWS EFS CSI Driver Operator with Secure Token Service
+
+This procedure explains how to configure the AWS EFS CSI Driver Operator with {product-title} on AWS Secure Token Service (STS).
+
+Perform this procedure after installing the AWS EFS CSI Operator, but before installing the AWS EFS CSI driver as part of _Installing the AWS EFS CSI Driver Operator_ procedure. If you perform this procedure after installing the driver and creating volumes, your volumes will fail to mount into pods.
```

Comment:
Can be "_Installing the AWS EFS CSI Driver Operator_"  a link to the proper chapter?

Reply:
No, unfortunately. We have constraints in being able to publish xrefs in module files. Once we have lifted the restriction, this could certainly be updated to use the link.

Reply:
The link is provided in the following Additional Resources section.